apis/pipeline/pipeline.yaml

```diff
@@ -3553,6 +3553,25 @@ paths:
                 default:
                     $ref: '#/components/responses/Error'
 
+        get:
+            operationId: ListNodePools
+            summary: List node pools
+            security:
+                - bearerAuth: []
+            tags:
+                - clusters
+            responses:
+                200:
+                    description: Node pools listed
+                    content:
+                        application/json:
+                            schema:
+                                type: array
+                                items:
+                                    $ref: '#/components/schemas/ListNodePoolsResponse'
```

Comment:
I think extracting this as `ListNodePoolsResponse` would be more correct.

Reply:
Fixed in https://github.com/banzaicloud/pipeline/compare/2ad4cafb067c442a1cfbf1b9cd4071e298b269c8..f640984ef5dee7ae5770cc55c9ccc9741c0e760b